index.html

```diff
@@ -0,0 +1,161 @@
+      Tournament Of Power hosted in the universe of DragonBall.
+    </p>
+    <h3 class="header-title-3">2022-11-11(FRI) ~ 13(SUN)</h3>
+    <h4 class="title-location">@CTICC (Cape Town International Convention Centre), GrandWest Arena, Cape Town South Africa</h4>
+  </section>
+
+  <section id="Program" class="container-fluid program-desktop">
+    <h3 class="title-name-2 container-fluid text-white text-center">Main Program</h3>
+    <div class="horizontal-line"></div>
+    <div class="container text-center">
+      <!-- program grid -->
+      <div class="program-grid">
+        <div class="program-card border-box">
+          <img class="program-image img-fluid" src="./assets/db-logo.png" alt="tournament of power logo"/>
+          <div class="program-words">
+          <h5 class="program-title mt-3">Warrior Ceremony</h5>
+          <p class="program-description">
+            Introduction of the warriors competing and the hosts for the event.
+          </p>
+        </div>
+        </div>
+        <div class="program-card border-box">
+          <img class="program-image img-fluid" src="./assets/db-logo.png" alt="tournament of power logo"/>
+          <div class="program-words">
+          <h5 class="program-title mt-3">Regulations</h5>
+          <p class="program-description">
+            Explaining the scoreboard and rules that will follow for the event.
+          </p>
+        </div>
+      </div>
+        <div class="program-card border-box">
+          <img class="program-image img-fluid" src="./assets/db-logo.png" alt="tournament of power logo"/>
+          <div class="program-words">
+          <h5 class="program-title mt-3">Match making</h5>
+          <p class="program-description">
+            Match making tag-team and solo event based on online rating.
+          </p>
+        </div>
+        </div>
+        <div class="program-card border-box">
+          <img class="program-image img-fluid" src="./assets/db-logo.png" alt="tournament of power logo"/>
+          <div class="program-words">
+          <h5 class="program-title mt-3">Guest interview</h5>
+          <p class="program-description">
+            Live back and forth open-conversations between guest speakers and audience.
+          </p>
+        </div>
+        </div>
+        <div class="program-card border-box">
+          <img class="program-image img-fluid" src="./assets/db-logo.png" alt="tournament of power logo"/>
+          <div class="program-words">
+          <h5 class="program-title mt-3">Raffle trivia</h5>
+          <p class="program-description">
+            Live trivia with a chance to win one of many exculsive prizes.
+          </p>
+        </div>
+        </div>
+      </div>
+      </div>
+      <button class="program-btn"><a class="program-link mt-5">SEE THE WHOLE PROGRAM</a></button>
+    <h3 class="speakers-title">Featured Speakers</h3>
+    <div class="speakers-horizontal"></div>
```

Comment:
- [ ] Please centre the `SEE WHOLE PROGRAM` button and  `Featured Speakers` heading for the mobile version.
### Your Implementation
![image](https://user-images.githubusercontent.com/45460192/190672479-f8899633-4969-4291-97d2-b33e5b8629f8.png)

Reply:
Solved.